python-sdk/tests_integration/sql/operators/test_load_file.py

```diff
@@ -1138,3 +1139,25 @@ def test_aql_nested_ndjson_file_to_database_explicit_illegal_sep_params(sample_d
     df = db.export_table_to_pandas_dataframe(test_table)
     assert df.shape == (1, 36)
     assert "payload_size" in df.columns
+
+
+def test_serialization_and_deserialization_of_table(sample_dag):
+    """
+    Test serialization and deserialization of table object
+    """
+    with sample_dag:
+        init_table = Table(
+            name="test", conn_id="gcp_conn", metadata=Metadata(schema="testing_region", region="east1")
+        )
+
+        @task(task_id="serialize")
+        def serialize():
+            return init_table
+
+        @task(task_id="verify")
+        def verify(t):
+            assert t == init_table
+
+        t = serialize()
+        verify(t)
+    test_utils.run_dag(sample_dag)
```

Comment:
@utkarsharma2 aas Kaxil mentioned. Ultimately we are just serializing data into json and then deserializing it from json. There is no need to create a DAG run and if anything that adds unnecessary overhead. Why not just serialize it into a json variable and ensure that the resulting json looks correct/can deserialize back to the same object?